storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3697,6 +3715,12 @@ static int rocksdb_init_func(void* const p)
     mysql_mutex_unlock(&rdb_sysvars_mutex);
   }
 
+  if (rocksdb_persistent_cache_size > 0) {
```

Comment:
To expand, it is also no trivial to determine if the cache size is valid so to save messy logic of user specifying one option and not the other (and I don't think its wise to check that user specified both size and path because then the cache could fail silently) I use the size as the flag, send values in and let rdb validate everything.